applications/FSIapplication/python_scripts/partitioned_fsi_base_solver.py

```diff
@@ -235,7 +235,7 @@ def _GetDomainSize(self):
 
     def _GetNodalUpdateUtilities(self):
 
-        structure_time_scheme = self.structure_solver.settings["scheme_type"].GetString()
+        structure_time_scheme = self.structure_solver.dynamic_settings["scheme_type"].GetString()
```

Comment:
Hi @rubenzorrilla 
I would recommend you to add a check here since you access the member of the solver directly.
I thought of sth like this:
~~~py
if self.structure_solver.Has(dynamic_settings["scheme_type"]):
    raise Exception("Dynamic settings not found")
~~~

Reply:
In any case, an error is thrown if there are no dynamic_settings. Am I wrong?